os/board/stm32l4r9ai-disco/src/stm32_boot.c

```diff
@@ -95,15 +95,16 @@ void board_initialize(void)
 #endif
 
 #ifdef CONFIG_STM32L4_LTDC
-    
     stm32_psram_initialize();
     printf("PSRAM init done\n");
     stm32_exflash_initialize();
     printf("NOR init done\n");
+#if 0	/* TODO */
```

Comment:
could you let me know why you comment out here?

Reply:
LCD init stucks here if I do clock configuration. STM should check this. Was communicated to STM in the mail chain